app/Http/Controllers/NetworkData/Online.php

```diff
@@ -18,7 +19,16 @@ public function getOnline()
                               },
                           ])->get();
 
+        $pilotSessions = Pilot::remember(2)
+            ->online()
+            ->withinDivision()
+            ->with([
+                'account' => function ($q) {
+                    $q->remember(1);
```

Comment:
I would suggest we ditch the account remembering, and instead just get the first and last name (for BOTH atc and pilot) `$q->get(['name_first', 'name_last']);`